login-workflow/docs/ContactSupport.md

```diff
@@ -0,0 +1,71 @@
+| contactSupportEmergency               | The text to override contact support emergency support    | `string`                                                          | no       | Emergency Support                                                           |
+| contactSupportTechnicalAssistance     | The text to override contact support technical assistance | `string`                                                          | no       | For technical support, please call 1-800-123-4567.                          |
+| ContactSupportIcon                    | A component to render the icon                            | `ReactNode`                                                       | no       | `<ChatBubbleOutline />`                                                     |
+| hideTitle                             | Prop to hide `title`                                      | `boolean`                                                         | no       | `false`                                                                     |
+| hideContactSupportContent             | Prop to hide `ContactSupportContent`                      | `boolean`                                                         | no       | `false`                                                                     |
+| hideContactSupportIcon                | Prop to hide `ContactSupportIcon`                         | `boolean`                                                         | no       | `false`                                                                     |
+| hideContactSupportQuestion            | Prop to hide `contactSupportQuestion`                     | `boolean`                                                         | no       | `false`                                                                     |
+| hideContactSupportMessage             | Prop to hide `contactSupportMessage`                      | `boolean`                                                         | no       | `false`                                                                     |
+| hideContactSupportEmergency           | Prop to hide `contactSupportEmergency`                    | `boolean`                                                         | no       | `false`                                                                     |
+| hideContactSupportTechnicalAssistance | Prop to hide `contactSupportTechnicalAssistance`          | `boolean`                                                         | no       | `false`                                                                     |
+| hideActions                           | Prop to hide `Actions`                                    | `boolean`                                                         | no       | `false`                                                                     |
```

Comment:
If you decide to go the object route with the hidden items here is the doc update as well
```suggestion
hiddenElements | Props to hide various elements of the component | ContactSupportHiddenElements | no | {}
```